chia/plotters/bladebit.py

```diff
@@ -60,37 +80,52 @@ def get_bladebit_package_path() -> Path:
     return Path(os.path.dirname(sys.executable)) / "bladebit"
 
 
-def get_bladebit_exec_venv_path() -> Optional[Path]:
+def get_bladebit_exec_path(with_cuda: Optional[bool] = None):
+    if with_cuda:
+        return "bladebit_cuda.exe" if sys.platform in ["win32", "cygwin"] else "bladebit_cuda"
+    return "bladebit.exe" if sys.platform in ["win32", "cygwin"] else "bladebit"
+
+
+def get_bladebit_exec_venv_path(with_cuda: Optional[bool] = None) -> Optional[Path]:
     venv_bin_path = get_venv_bin()
     if not venv_bin_path:
         return None
-    if sys.platform in ["win32", "cygwin"]:
-        return venv_bin_path / "bladebit.exe"
-    else:
-        return venv_bin_path / "bladebit"
+    bladebit_exec = get_bladebit_exec_path(with_cuda)
+    return venv_bin_path / bladebit_exec
 
 
-def get_bladebit_exec_src_path(plotters_root_path: Path) -> Path:
+def get_bladebit_exec_src_path(plotters_root_path: Path, with_cuda: Optional[bool] = None) -> Path:
     bladebit_src_dir = get_bladebit_src_path(plotters_root_path)
     build_dir = "build/Release" if sys.platform in ["win32", "cygwin"] else "build"
-    bladebit_exec = "bladebit.exe" if sys.platform in ["win32", "cygwin"] else "bladebit"
+    bladebit_exec = get_bladebit_exec_path(with_cuda)
     return bladebit_src_dir / build_dir / bladebit_exec
 
 
-def get_bladebit_exec_package_path() -> Path:
+def get_bladebit_exec_package_path(with_cuda: Optional[bool] = None) -> Path:
     bladebit_package_dir = get_bladebit_package_path()
-    bladebit_exec = "bladebit.exe" if sys.platform in ["win32", "cygwin"] else "bladebit"
+    bladebit_exec = get_bladebit_exec_path(with_cuda)
     return bladebit_package_dir / bladebit_exec
 
 
 def get_bladebit_executable_path(plotters_root_path: Path) -> Path:
-    bladebit_exec_venv_path = get_bladebit_exec_venv_path()
+    # Search for bladebit executable which supports CUDA at the first priority
+    bladebit_exec_venv_path = get_bladebit_exec_venv_path(with_cuda=True)
```

Comment:
Yes, regardless of the result of `is_cudaplot_available()`, `bladebit_cuda` is preferred over `bladebit` for now.
This can be changed as Bladebit development proceeds but for now this is the current spec.